honeybee_schema/energy/ventcool.py

```diff
@@ -103,3 +104,168 @@ class VentilationOpening(NoExtraBaseModel):
+        'in the VentilationSimulationControl object, a default value will be calculated to '
+        'produce air flow leakages equivalent to the  zone flow rate defined by the '
+        'corresponding Infiltration object.'
+    )
+
+    air_mass_flow_exponent_closed: float = Field(
+        default=0.7,
+        ge=0.5,
+        le=1,
+        description='An optional dimensionless number between 0.5 and 1 used to '
+        'calculate the mass flow rate when the opening is closed; required to run an '
+        'AirflowNetwork simulation. This value represents the leak geometry impact '
+        'on airflow, with 0.5 generally corresponding to turbulent orifice flow and 1 '
+        'generally corresponding to laminar flow. The default of 0.7 is representative '
+        'of internal and external window leakage, according to the DesignBuilder Cracks '
+        'template.'
+    )
+
+    minimum_density_difference_two_way: float = Field(
+        default=0.0001,
+        gt=0,
+        description='A number in kg/m3 indicating the minimum density difference above '
+        'which two-way flow may occur due to stack effect; required to run an '
+        'AirflowNetwork simulation. The default of 0.0001 is a typical default'
+        'value used for AirflowNetwork openings.'
+    )
+
+
+class AFNCrack(NoExtraBaseModel):
+    """Properties for airflow through a crack."""
+
+    type: constr(regex='^VentilationCrack$') = 'VentilationCrack'
+
+    air_mass_flow_coefficient_reference: float = Field(
+        ...,
+        gt=0,
+        description='A number in kg/s-m at 1 Pa per meter of crack length at the '
+        'conditions defined in the ReferenceCrack condition; required to run an '
+        'AirflowNetwork simulation. The DesignBuilder Cracks template defines the flow '
+        'coefficient for a tight, low-leakage wall to be 0.00001 and 0.001 for external '
+        'and internal constructions, respectively. Flow coefficients for a very poor, '
+        'high-leakage wall are defined to be 0.0004 and 0.019 for external and internal '
+        'constructions, respectively.'
+    )
+
+    air_mass_flow_exponent: float = Field(
+        default=0.65,
+        ge=0.5,
+        le=1,
+        description='An optional dimensionless number between 0.5 and 1 used to '
+        'calculate the crack mass flow rate; required to run an '
+        'AirflowNetwork simulation. This value represents the leak geometry impact '
+        'on airflow, with 0.5 generally corresponding to turbulent orifice flow and 1 '
+        'generally corresponding to laminar flow. The default of 0.65 is '
+        'representative of many cases of wall and window leakage, used when the '
+        'exponent cannot be measured.'
+    )
+
+    crack_factor: float = Field(
+        default=1,
+        gt=0,
+        le=1,
+        description='A number indicating multiplier for air mass flow through a crack.'
```

Comment:
It might be good to say what people use this factor to account for. For example, is it used to account for different shapes of cracks?

Reply:
@chriswmackey I just did some research and while there isn't any documentation for this factor, I think we can think of it as a way to linearly increment crack flows based on specific crack areas. 

For context, this parameter is exposed for both Window/Door openings (where it is called 'Opening Factor'), and for cracks is called the 'Crack Factor'. The EnergyPlus Engineering reference doesn't mention the crack factor at all. The input and output reference only describes the meaning of the value for windows/doors but not for the crack[1]. The description of Crack Factor as a multiplier only shows up in the IDF editor description. And the crack factor doesn't show up at all in the Hutcheon and Handergord text where the crack flow formula is derived. 

However, it's pretty easy to figure out what it does. The formula is basically: flow_rate = crack_factor * flow_coefficient * delta_pressure ** flow_exponent. So the crack factor allows you to linearly change the flow rate, without messing around with the other parameters. So if for some reason a user wants to account for a crack opening that is 50% smaller then the others, this would be the parameter to achieve that. Therefore the flow_exponent determines shape and sharpness of the orifice, and crack factor is to account for linear reduction of crack area.


[1] https://bigladdersoftware.com/epx/docs/9-3/input-output-reference/group-airflow-network.html#field-windowdoor-opening-factor-or-crack-factor

Reply:
Hm.  Without a clear use case where someone would specifically want to change the `crack_factor` instead of changing the `flow_coefficient`, I am inclined to keep the schema simpler and just use a single variable (the `flow_coefficient`) rather than having two variables that can do effectively the same thing.  I know that the IDF schema is kinda notorious for having multiple inputs that correspond to the same thing and I have often found that managing the multiple variables can make it harder to figure out what the value actually is in the simulation (in this case, figuring out what the flow rate is). Granted, I can see why someone might want the extra `crack_factor` input if they are manually editing IDF files in a text editor and they want to remember their previous `flow_coefficient` as they progressively reduce the infiltration through a crack. But for people who will be producing honeybee schema files from a script, it just feels like we are better off without the extra complication to the two variables.

So what do you think about just leaving out the `crack_factor` from the schema for now unless we see someone raising a specific use case where they really need it?

Reply:
I agree with your logic, let's remove it.